src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -445,5 +445,35 @@ internal static void PropagateResourcesChanged(object instance, ResourceUpdateRe
 				}
 			}
 		}
+
+		private static string GetCommandLineArgsWithoutExecutable()
+		{
+			var args = Environment.GetCommandLineArgs();
+			if (args.Length < 1)
```

Comment:
```suggestion
			if (args.Length <= 1)
```
no point in continuing with the rest if there just executable name/path, right?